src/main/java/org/folio/services/group/GroupFiscalYearTotalsService.java

```diff
@@ -82,11 +84,13 @@ public Future<GroupFiscalYearSummaryCollection> getGroupFiscalYearSummaries(Stri
         }))
       .compose(holders -> updateHoldersWithAllocations(holders, requestContext)
         .map(holder -> {
+          log.info("getGroupFiscalYearSummaries:: Updating group summary with allocation fields for '{}' holder(s)", holders.size());
           updateGroupSummaryWithAllocation(holders);
           return null;
         })
         .compose(v -> updateHoldersWithTransfers(holders, requestContext))
         .map(v -> {
+          log.info("getGroupFiscalYearSummaries:: Updating group summary with calculated fields for '{}' holder(s)", holders.size());
```

Comment:
these 2 logs should be with debug level as for me

Reply:
changed to debug level, thank you